test/integration/scheduler/taint_test.go

```diff
@@ -665,6 +667,7 @@ func TestTaintBasedEvictions(t *testing.T) {
 			podTolerations.SetExternalKubeInformerFactory(externalInformers)
 
 			context = initTestScheduler(t, context, true, nil)
+			defer cleanupTest(t, context)
```

Comment:
we should call this for the other test above (TestTaintBasedEvictions).

Reply:
You mean `TestTaintNodeByCondition` above?

Reply:
yes, sorry for the confusion.

Reply:
it's now covered in https://github.com/kubernetes/kubernetes/commit/cd675cca922585677d58413fae93296ea307a916